doc/error-handling.md

```diff
@@ -17,22 +17,23 @@ The logger is thread-safe and can hence be safely used from multiple threads con
 
 ## Log Levels
 
-The following log levels are supported, ordered by criticality from lowest to highest.
+The following log levels are supported, ordered by the ammount of information displayed from highest to lowest.
```

Comment:
amount*

Reply:
Yes...args I cannot believe I make mistakes like this. :-) Anyway, sure, will change it.